### PR TITLE
Update .travis.yml file

### PR DESCRIPTION
**Pull request by zzet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,11 @@ sudo: false
 language: ruby
 rvm:
   - 2.3.3
+  - 2.3.6
```
Comment (ivanovaleksey) [inline .travis.yml]: Not sure if 2.3.3 should be tested separately.
I believe it was just latest 2.3 version at the moment

Comment (zzet) [reply .travis.yml]: Fixed

**Comment by ivanovaleksey** [comment]
It would be better to split Rubocop and Travis into separate PRs if it's possible.

**Comment by zzet** [comment]
если бы код в мастере проходил тесты - то имело бы место такое разделение. А раз не проходит - должно быть пофикшено asap

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.3
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
 
-before_install: gem install bundler -v 1.13.6
+before_install: gem install bundler -v 1.16.1
```
Comment (ivanovaleksey) [inline .travis.yml:8]: Actually I don't think we should preinstall bundler manually at all.

**Pull request closed without merging**